assessments/0002-notary-project.md

```diff
@@ -0,0 +1,176 @@
+- V1 content seems quite good, but looks like it was developed for a site that no longer uses it.
+- For MVP, I'd suggest focusing on V2 content, and linking out to the existing V1 documentation in the location that is currently rather than incorporating it right off the bat.
+
+Eventually I'd expect that V1 would be moved to the Notary Project org, but that is a project level consideration. Even if the whole repo doesn't get moved, there may be value in moving the docs out of TUF's notary repo and into the notaryproject/notaryproject.dev repo.
+
+## Contributor documentation 
+
+**Note**, as my recommendation above is to link to the V1 documentation as is for the website MVP, the remainder of this assessment will only be looking at the existing V2 documentation.
+
+Criteria 
+- 1 = (Is not present or requires significant work)
+- 3 = (is present, but needs work)
+- 5 = (is executed extremely well or no improvement required) |
+
+| Criteria                                  | 1   | 2   | 3   | 4   | 5   |
+| ---                                       | --- | --- | --- | --- | --- |
+| Communication methods documented          |     |     | ✅  |     |     |
+| Beginner friendly issue backlog           |     | ✅  |     |     |     |
+| “New contributor” getting started content | ✅  |     |     |     |     |
+| Project governance documentation          | ✅  |     |     |     |     |
+
+### Comments
+
+_Provide comments for each rating above, 1-2 sentences max, bullet point list_
+
+**Communication methods documented**
+- There is a Contributing & Conversations section in the main README (tho, it is mis-linked in the table of contents)
+
+**Beginner friendly issue backlog**
+- There are "Documentation," "spec," and "good first issue" issue tags
+- Most issues seem to link out to hackmd.io notes, but those note are are well written and informative
+
+**“New contributor” getting started content**
+- Same as above, there is a Contributing & Conversations section, but there are not yet any instructions for contributions.
+
+**Project governance documentation**
+- This appears to be actively being worked on, and there is a "Governance" issue tag in use. 
+
+
+### Recommendations
+
+Most of the issues in this section are a function of not having a website yet, and the project still being in a building phase.
+- As we go through the site build project, we should put together a skeleton IA that can be filled in as the project grows.
+
+Regarding beginner friendliness, as we move through the project, we should consider processes/documents for:
+* Triaging docs issues
+* Clearly marking a way for new contributors to make code or documentation contributions (i.e. a “good first issue” label), and define what makes a good first issue
+* Maintaining issues for staleness
+* New contributors/your first contribution
+* Letting new users know where to get help
+
+
+## Website
```

Comment:
Given that no website exists, I don't think you need to go into too much detail other than saying "No website exists and therefore we will not evaluate this section". And then literally don't fill it in!